docs/setup/deployment.md

```diff
@@ -56,4 +56,4 @@ kubectl port-forward ${POD} 9889 9080
 ```
 
 `9889` is the default port for Web UI, `9080` is the default port of scheduler's Restful service where web UI retrieves info from.
-Once this is done, web UI will be available at: http://localhost:9889.
\ No newline at end of file
+Once this is done, web UI will be available at: http://localhost:9889.
```

Comment:
what has changed on this line?

Reply:
This was unintentional. I checked the two files and I introduces a newline by mistake. I will remove it.

Reply:
This was unintentional. I checked the two files and I introduces a newline by mistake. I will remove it.